_rules/SC4-1-1-aria-valid.md

```diff
@@ -0,0 +1,114 @@
+---
+name: Validity of ARIA attribute specified
+
+description: |
+ This rule checks that the `aria-*` attribute specified is valid.
+
+success_criterion:
+- 4.1.1
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+Any DOM element within the `body` of a webpage with an `aria-*` attribute.
+
+### Expectation
+
+The `aria-*` attribute specified is a valid attribute.
```

Comment:
This should start with an "Each target element ...". It's also ambiguous about what it means for an aria attribute to be valid. I think what this should say is that all attributes that start with `aria-` should attributes defined in ARIA 1.1.